src/_component/skills/skills.scss

```diff
@@ -0,0 +1,73 @@
+.skills-list { 
+    border-radius: 6px;
+    border:2px solid #707070 ;
+    list-style-type: none;
+}
+
+.skills__item {
+    display: flex;
+    border:2px solid #707070 ;
+    background-color: white;
+
+    &:first-child .skills__icon { 
+        color: #D52027;
+    }
+    &:nth-child(2) .skills__icon { 
+        color: #50D166;
+    }
+
+     &:last-child .skills__icon { 
+         color: #F18F1C;
+    }
+     
+}
+
+.skills__count,
+.skills__icon {
+    padding: 10px 12px 8px 10px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}   
+
+.skills__icon,
+// .skills__count,
+.skills__label { 
+    line-height: 48px;
+}
+
+.skills__label { 
+    flex-grow: 1;
+    color: #999999;
+}
+
+.skills__count {
+    color: #666666 ;
+    font-size: 18px;
+}
+
+
+.skills_skill--heart { 
+    background-color: #D52027;
+    ;
```

Comment:
Probably you forgot to delete this ";" ?

Reply:
yes, you are right :D tnx